c7n/resources/emr.py

```diff
@@ -332,6 +333,52 @@ def augment(self, resources):
         return resources
 
 
+@EMRSecurityConfiguration.filter_registry.register('kms-key')
+class EmrSecurityConfigurationKmsFilter(KmsRelatedFilter):
+    """
+    Filter a resource by its associcated kms key and optionally the alias name
+    of the kms key by using 'c7n:AliasName'
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: emr-security-configuration-kms-key
+                resource: aws.emr-security-configuration
+                filters:
+                  - type: kms-key
+                    key: c7n:AliasName
+                    value: "^(alias/aws/)"
+                    op: regex
+    """
+
+    RelatedIdsExpression = 'SecurityConfiguration.EncryptionConfiguration.AtRestEncryptionConfiguration.\
+                            LocalDiskEncryptionConfiguration.AwsKmsKey'
+
+    related_ids = []
+
+    def get_related_ids(self, resources):
+        if not self.related_ids:
+            alias_names = set(jmespath.search(
```

Comment:
style nit: can dedent this block with if self.related_ids: return self.related_ids